docs/getting-started/prerequisites.md

```diff
@@ -34,13 +34,19 @@ cd clusterfuzz
 Install the Google Cloud SDK by following the instructions
 [here](https://cloud.google.com/sdk/).
 
-Once this is done, run:
+### (Optional) Log in to your Google Cloud account
+If you are planning to set up ClusterFuzz in [production], you should
+authenticate your account with the `gcloud` tool. This is **not** necessary if
+you are simply running ClusterFuzz [locally].
```

Comment:
Maybe add "To log in, run:"?

Reply:
Didn't like adding this repetition, so I reordered the local/production bits to make this clearer.